xml/System/Math.xml

```diff
@@ -1021,7 +1021,7 @@
         <param name="y">The y coordinate of a point.</param>
         <param name="x">The x coordinate of a point.</param>
         <summary>Returns the angle whose tangent is the quotient of two specified numbers.</summary>
-        <returns>An angle, θ, measured in radians, such that -π ≤ θ ≤ π, and tan(θ) = <paramref name="y" /> / <paramref name="x" />, where (<paramref name="x" />, <paramref name="y" />) is a point in the Cartesian plane. Observe the following:
```

Comment:
This is not a valid change because of requirements put in place by the IEEE 754 (Chapter 9) and C Language (Annex F) Specifications:
* `atan2(±0, −0)` returns `±π`.
* `atan2(±0, x)` returns `±π` for `x < 0`.
* `atan2(±y, −∞)` returns `±π` for `finite y > 0`.

There are all special values that are not in the normal input domain and where the result returned is considered "inexact".